### PR TITLE
draft ms graph shipping instructions

### PR DESCRIPTION
**Pull request by imnotashrimp**
### What changed

Added a log shipping page for Microsoft Graph API. 

## Pages to review

- [x] https://deploy-preview-272--logz-docs.netlify.com/shipping/ (click the **Azure** tag)
- [x] https://deploy-preview-272--logz-docs.netlify.com/shipping/log-sources/microsoft-graph.html

**Notes for reviewers**:

@yyyogev some notes for you here - 

- One gap: There's a note that says `If the Grant admin consent for Default Directory button is disabled, contact an Azure administrator for your organization.`
  We should be more specific here - what does the user need to ask for from the admin? Does the admin need to enable this option separately? Does the user need to be granted admin access?
- I removed reference to the sample config and made it a code block instead—it's short and can be referenced directly on the page. Happy to hear if this was the wrong thing to do :)
- I combined all the parameters into one list. Again, not sure if this was the best move. Read through it and let me know what you think.

## Remaining work

- [x] Technical review
- [x] Copy Review

## Post launch

**Do not remove** - To be completed by the docs team upon merge:
- [ ] convert & copy to the logzio/microsoft-graph repo readme

### CODE REVIEW

**Comment by yyyogev** [comment]
@imnotashrimp looks good.
As for the admin consent, only an admin can click this button. So the user needs either for admin permission or to ask an admin to click it for him.

I'm ok with the sample config file in the doc, let's just add `debug: INFO` to it

I see you removed my note that said the config file at the docker end must be named config.yaml
Do you think it's clear enough?

**Comment by imnotashrimp** [comment]
@yyyogev Revised in latest push.

> As for the admin consent, only an admin can click this button. So the user needs either admin permission or to ask an admin to click it for him.

Added it to the note

> I'm ok with the sample config file in the doc, let's just add `debug: INFO` to it

Done

> I see you removed my note that said the config file at the docker end must be named config.yaml
> Do you think it's clear enough?

In your experience, do folks modify the command without making sure the code supports it? I prefer to remove it, but can add it back in if you think it's important enough.

File: _source/logzio_collections/_log-sources/microsoft-graph.md [hunk 1]
```diff
@@ -72,8 +72,8 @@ please [open an issue](https://github.com/logzio/microsoft-graph/issues/new) at
 
     Click **Grant admin consent for Default Directory > Yes**.
 
-    If the _Grant admin consent for Default Directory_ button is disabled,
-    contact an Azure administrator for your organization.
+    Only Azure administrators can change the _Grant admin consent for Default Directory_ setting.
```
Comment (yyyogev) [inline _source/logzio_collections/_log-sources/microsoft-graph.md]: Perhaps 'only azure admins can grant consent for default directory...?

Comment (imnotashrimp) [reply _source/logzio_collections/_log-sources/microsoft-graph.md]: Revised, please take a look

Comment (yyyogev) [reply _source/logzio_collections/_log-sources/microsoft-graph.md]: Love it

**Comment by yyyogev** [comment]
@imnotashrimp can you please run your magic script and update the integration repos readme?
thanks

**Comment by imnotashrimp** [comment]
> @imnotashrimp can you please run your magic script and update the integration repos readme?
> thanks

Of course! Today's the day

**Comment by imnotashrimp** [comment]
@yyyogev Please approve this PR so I can merge.

**Pull request merged**